src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -176,6 +173,22 @@ def _get_current_exposure(self, index: pd.Index) -> pd.Series:
 
 
 class DynamicChildWastingModel(DiseaseModel):
+    @property
+    def columns_created(self) -> List[str]:
+        return [self.state_column]
+
+    @property
+    def columns_required(self) -> Optional[List[str]]:
+        return ["age", "sex", "static_child_wasting_propensity"]
+
+    @property
+    def initialization_requirements(self) -> Dict[str, List[str]]:
+        return {
+            "requires_columns": ["age", "sex", "static_child_wasting_propensity"],
+            "requires_values": ["static_child_wasting.exposure"],
```

Comment:
Does it actually require this pipeline? It wasn't declared before.

Reply:
Sim runs fine without it. Removed.